File: app/src/main/java/com/capstone/bankit/data/models/LoginResponse.java
package com.capstone.bankit.data.models;

public class LoginResponse {
    private int statusCode;
    private String message;
    private Data data;

    // Getters and setters
    public int getStatusCode() {
        return statusCode;
    }

    public void setStatusCode(int statusCode) {
        this.statusCode = statusCode;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public Data getData() {
        return data;
    }

    public void setData(Data data) {
        this.data = data;
    }

    public static class Data {
        private String customToken;

        public String getCustomToken() {
            return customToken;
        }

        public void setCustomToken(String customToken) {
            this.customToken = customToken;
        }
    }
}

File: app/src/main/java/com/capstone/bankit/data/models/LoginResponse.kt
package com.capstone.bankit.data.models

import com.google.gson.annotations.SerializedName

data class LoginResponse(

	@field:SerializedName("data")
	val data: DataToken? = null,

	@field:SerializedName("message")
	val message: String? = null,

	@field:SerializedName("statusCode")
	val statusCode: Int? = null
)

data class DataToken(

	@field:SerializedName("customToken")
	val customToken: String? = null
)
